### PR TITLE
feat(server): Remove default JWT_SECRET value in .env

### PR DESCRIPTION
**Pull request by bo0tzz**

### CODE REVIEW

**Review by zackpollard — APPROVED** [review]

File: docker/.env.example [hunk 1]
```diff
@@ -38,7 +38,10 @@ LOG_LEVEL=simple
 # JWT SECRET
 ###################################################################################
 
-JWT_SECRET=randomstringthatissolongandpowerfulthatnoonecanguess
+# This JWT_SECRET is used to sign the authentication keys for user login
+# You should set it to a long randomly generated value
+# You can use this command to generate one: openssl rand -base64 128
+#JWT_SECRET=
```
Comment (alextran1502) [inline docker/.env.example:44]: I would remove the comment so that it should as something you should populate
```suggestion
JWT_SECRET=
```

Comment (zackpollard) [reply docker/.env.example:44]: If it's left blank, what will happen on the server, will it fail to start?

Comment (alextran1502) [reply docker/.env.example:44]: > Will the server properly error on an empty value?

Hmm need to check that.

Comment (alextran1502) [reply docker/.env.example:44]: image

We are good to go with the suggestion

**Comment by alextran1502** [comment]
We also need to update the [install.sh](https://github.com/immich-app/immich/blob/main/install.sh) script in the `populate_env_file` environment.

**Pull request merged**